.github/workflows/gem-push.yml

```diff
@@ -0,0 +1,32 @@
+name: Ruby Gem
+
+on:
+#   push:
+#     branches: [ master ]
+  # run manually for now
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build + Publish
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.6.x
+
+    - name: Publish to RubyGems
+      run: |
+        mkdir -p $HOME/.gem
+        touch $HOME/.gem/credentials
+        chmod 0600 $HOME/.gem/credentials
+        printf -- "---\n:rubygems_api_key: ${GEM_HOST_API_KEY}\n" > $HOME/.gem/credentials
```

Comment:
Will this log the token to the Action's artifacts? If so, can it be used maliciously?

Reply:
Its a push only API token and you can't replace an existing gem version in rubygems.org.  This should not end up in artifacts from what I understand.  This is literally the supplied code from GitHub for implementing this action.

Reply:
The above line puts the credentials in a file, it won't be logged/printed anywhere.

Reply:
Ah! Thanks for the explanation! That was the piece I was missing.